### ISSUE TITLE
Model Loss in Test Stage

### ISSUE THREAD

**Issue by DocF** [csuhan/s2anet#28]
First, thx for this  excellent repo.
How could I print loss in test stage.
I changed `result = model(return_loss = False, xxxxx)` to`result = model(return_loss =True, xxxxx)`  in line 31 of test.py. But run error.
looking forward to your reply.

**Comment by csuhan** [comment]
To my knowledge, you cannot get the loss during testing because mmdet will start a training process if you set `return_loss=True`.
Please refer to: [https://github.com/csuhan/s2anet/blob/173ef2151a8163148f414bc65423d9a9f187961a/mmdet/models/detectors/base.py#L98](https://github.com/csuhan/s2anet/blob/173ef2151a8163148f414bc65423d9a9f187961a/mmdet/models/detectors/base.py#L98)

**Comment by DocF** [comment]
Ok, got it. Thx for your reply.
